test/dotnet-new3.UnitTests/DotnetNewInstall.cs

```diff
@@ -500,5 +500,24 @@ public void ReinstallDoesntRemoveTemplates()
                 .And.HaveStdOutContaining("TestAssets.TemplateWithTags")
                 .And.HaveStdOutContaining("TestAssets.ConfigurationKitchenSink");
         }
+
+        [Fact]
+        public void CannotInstallTemplateWithoutMandatoryConfig()
+        {
+            var home = TestUtils.CreateTemporaryFolder("Home");
+            string invalidTemplatePath = TestUtils.GetTestTemplateLocation("Invalid/MissingMandatoryConfig");
+            new DotnetNewCommand(_log, "-i", invalidTemplatePath)
+                .WithCustomHive(home)
+                .WithWorkingDirectory(TestUtils.CreateTemporaryFolder())
+                .Execute()
+                .Should()
+                .ExitWith(0)
+                .And.NotHaveStdErr()
```

Comment:
nit: I prefer to have different order of this 2 commands:
```
.And.NotHaveStdErr()
.ExitWith(0)
```
So when it fails it prints what StdErr is... Probably `ExitWith` could be changed to print `StdErr` when it fails...

Reply:
`ExitWith` prints `StdErr` via `AppendDiagnosticsTo`